include/sparrow/layout/primitive_array.hpp

```diff
@@ -325,14 +380,23 @@ namespace sparrow
                 return true;
             }
         );
-        return self_type::create_proxy(std::forward<R>(range), std::move(iota_to_is_non_missing));
+        return self_type::create_proxy(
+            std::forward<R>(range),
+            std::move(iota_to_is_non_missing),
+            std::forward<std::optional<std::string_view>>(name),
+            std::forward<std::optional<std::string_view>>(metadata)
```

Comment:
No need for `std::forward` here, prefer `std::move`.

Reply:
done